inquirer/render/console/base.py

```diff
@@ -1,13 +1,10 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import print_function
+from math import ceil, floor
 
```

Comment:
Why do you import `ceil` if you are using only floor?

Reply:
Not sure TBH I have removed it in the latest commit